_drafts/newsletter-003.md

```diff
@@ -297,57 +297,57 @@ follow [@logicsoup] on Twitter.
 [garden-october]: http://cyberplant.xyz/posts/october
 [@logicsoup]: https://twitter.com/logicsoup
 
-### [Godot][godot] and Rust
-
-- Rendering a 2D game in 3D
-
-  **TODO**: img
-
-  <https://medium.com/@recallsingularity/rendering-a-2d-game-in-3d-bd24ddbee6eb>
-
-  <https://twitter.com/RecallSingular1/status/1184059000242855936>
+### [EVE Online] Backend Optimization
```

Comment:
This doesn't seem to come from the Eve Online team. Are we sure it's used in production?

Reply:
Renamed